Update usaco-966.mdx with cpp code (#3026)
cpinitiative/usaco-guide: solutions/usaco-966.mdx

print(ans)
```
</PySection>
<CPPSection>

```cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
	int n;
	cin >> n;
        
	/*
	 * Lists the first 8. 
	 * 14 is in the zeroth position since if were trying to find the 
	 * eighth number, 8 % 8 = 0.
	 */

> 	vector <int> v{14, 1, 2, 4, 7, 8, 11, 13};

Reviewer: ```suggestion
	vector<int> v{14, 1, 2, 4, 7, 8, 11, 13};
```

> 	vector <int> v{14, 1, 2, 4, 7, 8, 11, 13};

Author: ```suggestion
	vector<int> v{14, 1, 2, 4, 7, 8, 11, 13};
```

	
	int ans = 0;
	// Finds the group
	ans = (n - 1) / 8;
	

> 	// Distance away n is from the first 8

Author: ```suggestion
      /*
       * Get the distance away N from the first 8  by calculating (N/8)*15  , since we 
       * have to add for every group of 15.
       */

```

> 	// Distance away n is from the first 8

Author: ```suggestion
	//Group every 15 by calculating answer(i.e(N/8)*15)
```

> 	// Distance away n is from the first 8

Reviewer: what does this mean

Author: > what does this mean

Is there any mistake in the comment?

> 	//Group every 15 by calculating answer(i.e(N/8)*15)

Reviewer: ```suggestion
	//Since every 15 numbers are a group...
```

> 	//Group every 15 by calculating answer(i.e(N/8)*15)

Author: ```suggestion
	//Group every 15 numbers are a group calculate answer
```

	ans *= 15;
	

> 	// Adds the corresponding element

Reviewer: ```suggestion
	// Adds the corresponding leftover
```

> 	// Adds the corresponding element

Author: ```suggestion
	// Adds the corresponding leftover
```

print(ans)
```
</PySection>
<CPPSection>

```cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
	// your code goes here
	int n;
	cin >> n;

>        // Make a vector/array of the first 8. 

Author: ```suggestion
        // Make a vector/array of the first 8. 
```

>         // Make a vector/array of the first 8. 

Author: ```suggestion
       // Make a vector/array of the first 8. 
```

>          // Make a vector/array of the first 8. 

Author: ```suggestion
        // Make a vector/array of the first 8. 
```

>          //14 is in the zeroth position    

Author: ```suggestion
        //14 is in the zeroth position    
```

> 	// vector the first 8. 
>     //14 is in the zeroth position     since if were trying to find the 

Author: ```suggestion
     // vector the first 8. 
    //14 is in the zeroth position    
    // Since if were trying to find the eighth number, it will be 8%8=0
```

>     //eighth number, 8 % 8 = 0.

Reviewer 2: indentation is weird here

Author: > indentation is weird here

Okay I will make a change

Author: Yes I will indent again

Author: Yes I resolved it

>     // Since if were trying to find the eighth number, it will be 8%8=0

Author: ```suggestion
         // Since if were trying to find the eighth number, it will be 8%8=0
```

>          // Since if were trying to find the eighth number, it will be 8%8=0

Author: ```suggestion
        // Since if were trying to find the eighth number, it will be 8%8=0
```

print(ans)
```
</PySection>
<CPPSection>

```cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
	// your code goes here
	int n;
	cin >> n;

> 	

Author: ```suggestion
```

>      // vector the first 8. 

Author: ```suggestion
         // Make a vector/array of the first 8. 
```

>     //14 is in the zeroth position    

Author: ```suggestion
           //14 is in the zeroth position    
```

>            //14 is in the zeroth position    

Author: ```suggestion
         //14 is in the zeroth position    
```

    // Since if were trying to find the eighth number, it will be 8%8=0

>     //eighth number, 8 % 8 = 0.

Author: ```suggestion
 
```
